detail using cylc message to add to scheduler log (#603)
cylc/cylc-doc: src/tutorial/furthertopics/message-triggers.rst

* Refer to these messages in the ``[dependencies]`` section of our workflow.

> .. note::
> 
>    Messages are recorded in the Workflow's scheduler log. Make sure that
>    they convey useful information for users reading the log file. See
>    :ref:`scheduler logs.cylc message` for details of how messages appear in
>    the log file.
> 

Reviewer: ```suggestion
.. note::

   The message will be recorded in the workflow's scheduler log.
   See :ref:`scheduler logs.cylc message` for details of how messages appear.

```
